tardis/io/schemas/debug.yml

```diff
@@ -0,0 +1,37 @@
+$$target: debug.yml
+type: object
+additionalProperties: false
+properties:
+  logging_state:
+    type: boolean
+    default: False
+    description: Logging Status for the Simulation execution
+  logging_level:
+    type: string
+    default: "Info"
+    description: Sets the Logging Level for the logger
+    properties:
+      type:
+        enum:
+          - "Notset"
+          - "Info"
+          - "Debug"
+          - "Warning"
+          - "Error"
+          - "Critical"
+  # debug_packets:
```

Comment:
Are these comments needed?

Reply:
No, I was just testing it. This is a draft PR right now 🙃

Reply:
These properties will be eventually moved to this Schema for having one place for all the Debugging related setups 😉